File: Modul_4/Bahasa_Python/PRAK402-2510817120005-NorKhalisyah.py
n= int(input("Masukkan batas maksimal: "))

for i in range(1, n + 1):
    if i % 2 != 0:
        print(i, end=" ")
print()

for i in range(n, 1, -1):
    if i % 2 == 0:
        print(i, end=" ")

File: Modul_4/Bahasa_C/PRAK402-2510817120005-NorKhalisyah.c
#include <stdio.h>
int main () {

    int n;
    printf("Masukkan batas maksimal: ");
    scanf("%d", &n);

    for (int i = 1; i <= n; i++) {
        if (i % 2 != 0) {
            printf("%d ", i);
    }
}
printf("\n");

    for (int i = n; i >= 2; i--) {
        if (i % 2 == 0) {
            printf("%d ", i);
    }
}

    return 0;
}